compose.yaml

```
@@ -0,0 +1,18 @@
+    ports:
+      - "127.0.0.1:8000:8000"
+
+  frontend:
+    image: HACK/frontend
+    build: ./client
+    volumes:
+      - ./client:/app/server
+    ports:
+      - "127.0.0.1:5173:5173"
+    links:
+      - backend
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)